### DISCUSSION TITLE
在windows上有没有办法编译完之后将add_packages的dll自动拷贝到编译目录下

### DISCUSSION THREAD

**Discussion by supf1994** [xmake-io/xmake#3234] [Q&A]
### 你在什么场景下需要该功能？

windows系统下需要该功能方便部署

### 描述可能的解决方案

有没有办法实现类似vcpkg那样再编译完之后将add_packages的dll自动拷贝到编译目录下

**Comment by xq114** [comment]
xmake install -o dist打包部署；编译后要试运行直接xmake run xxx

**Comment by waruqi** [comment]
安装时候会 copy
